docs/Console.md

```diff
@@ -0,0 +1,184 @@
+# Console
+
+In this tutorial, we will be building a console application that does basic console logging.
+
+---
+
+Create a .NET Core 2.0 (or above) console application named "ConsoleLoggingTutorial".
+
+---
```

Comment:
For line 7 and line 11, should there be any code snippets? If not, I think it looks a little funky with the horizontal rules